documents/coredocument.go

```diff
@@ -158,6 +165,16 @@ func newCoreDocument() (*CoreDocument, error) {
 	}, nil
 }
 
+func newFingerprint() *TransitionRulesFingerprint {
+	f := coredocumentpb.TransitionRulesFingerprint{
+		Roles:           nil,
+		TransitionRules: nil,
+	}
+	return &TransitionRulesFingerprint{
```

Comment:
Following on the comment above, couldnt we just return the coredoc. TransitionRulesFingerprint object?